content/blog/2019-02-19-Incubator.md

```diff
@@ -0,0 +1,59 @@
+
+[ESIP Incubator](https://www.esipfed.org/esip-lab/funding-opportunities). Moves from Stage 1 to Stage 2 by uniting teams around technology. Provides incentives for collaboration and community-building activity.
+
+[Mozilla Open Leaders](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/). Targets people with projects at any Stage including prior to  Stage 0. Teaches them open leadership skills that are helpful through to Stage 3. A typical graduate is able to recruit ~2 or 3 collaborators at a hackathon. (Moves one Stage forward during the program, often Stage 0 to 1, with skills to continue moving towards Stage 3)
+
+[NumFOCUS](https://numfocus.org/). Exists at Stage 4 for fiscally sponsored projects. Would like incubator to bring projects from 2 to 3, to be included as 4. Provides events planning, fundraising, for communities.
+
+This raises the question: which stages should a URSSI incubator be focused on?  What stages of a project would enter an URSSI incubation process? What does "graduating" from URSSI incubation mean?
+
+## How an incubator could work
+
+Graduating a project from one Stage to another is not trivial. Each stage requires different resources and expertise. An NSF funded incubator could help research software along the path by first credentialing project based on the stage that they have attained, and then offering mentorship or workshops to give teams the necessary skills to advance to the next stage.
+
+Successful sustainability is not guaranteed for every scientific research software project. Realistically, most projects fail. This raises the important question of which projects an Incubator would invest in, and how it would decide between them, given limited resources.
+
+One way to address this resource scarcity is for an NSF incubator to leverage its NSF affiliation as a source of incentives. NSF proposals already evaluate applications based on “broader impact” and “intellectual merit”. What if contributions to research software were considered for their broader impact and intellectual merit? What if the amount of impact and merit of these contributions depended on the stage of the software project? For example, a contribution to a project at Stage 2 (which is used by multiple institutions) could be considered to have more impact and merit than a project at Stage 1. An URSSI Incubator could support research software by adjusting the incentives around its development. By credentialing projects as having achieved certain Stages and changing the criteria for grant acceptance to take these stages into account, that would orient scientific research communities towards greater cooperation on software.
+
+## What about the commercial option?
+
+We acknowledged, but did not discuss in depth, a different path for sustainability for research software: proprietary commercialization. The NSF supports this already through NSF I-Corps supports this. 
```

Comment:
The NSF supports this already through NSF I-Corps supports this.  -> The NSF supports this already through NSF [I-Corps](https://www.nsf.gov/i-corps).